BeeSwift/GoalViewController.swift

```diff
@@ -475,6 +477,12 @@ class GoalViewController: UIViewController, UITableViewDelegate, UITableViewData
         }
     }
     
+    func setInitialCommentTextField() {
+        guard let lastDatapoint = self.goal?.recent_data?.last as? JSON, let comment = lastDatapoint["comment"].string else { return }
```

Comment:
easy enough
the same is done for the value

but what if one were to delete the comment each time? sure the clearButton is being shown but could mean an extra tap